src/ADNLPProblems/hs378.jl

```diff
@@ -0,0 +1,18 @@
+export hs378
+
+function hs378(; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
+  A = T[-6.089,-17.164,-34.054,-5.914,-24.721,-14.986,-24.1,-10.708,-26.662,-22.179]
```

Comment:
```suggestion
  A = [-6089 // 1000,-17164 // 1000,-34054 // 1000,-5914 // 1000,-24721 // 1000,-14986 // 1000,-241 // 10,-10708 // 1000,-26662 // 1000,-22179 // 1000]
```
this way we don't need to convert the data.